### ISSUE TITLE
New category: Robot/Artificial voice

### ISSUE THREAD

**Issue by RubenKelevra** [ajayyy/SponsorBlock#1321]
I noticed that more and more videos are using artificial voices (I guess due to their creators feel like they have a bad accent or not enough practice speaking English).

It would be nice if videos could be flagged as a whole when this type of voice over is used.

**Comment by ajayyy** [comment]
I don't think this is suitable for the current category system

**Comment by SameenAhnaf** [comment]
Why is this category required? Does it matter whether a video has artificial voiceover or not?

**Comment by RubenKelevra** [comment]
> Why is this category required? Does it matter whether a video has artificial voiceover or not? 

Well, yes. You can see from the votes (before YouTube removed downvotes) that they are not perceived as equal to human voice-overs.

They are usually very low quality in terms of intonation and I usually skip them when I encounter them as its super hard to follow the robotic voice any length of time. That's why I think this could be an interesting category for this plugin as well.

I know that there are good "AI speaker" channels out there, but I haven't found a single good "voice-over" channel yet. They are 100% bad up to absolutely terrible.

Example:

https://youtu.be/m3yu5SnNdsE

Where half of the comments make fun of the bad narration.

**Comment by sancarn** [comment]
Doesn't seem to fit... Generally whole videos use artificial voices, rather than parts of videos. So it doesn't really fit the categorisation of SponsorBlock imo

**Comment by QmwJlHuSg9pa** [comment]
Piggybacking on this issue: I want to block or flag videos that use particular hash tags (as well as hide them from search results, if possible), e.g. "hdremastered" (as popularised by Vevo). I can find no official way of achieving this within YouTube.

Yes, probably out of scope for SponsorBlock, but can anyone suggest a workable solution?

**Comment by ajayyy** [comment]
check out Blocktube
